Python/Product/PythonTools/PythonTools/Project/AddVirtualEnvironment.xaml

```diff
@@ -53,13 +52,13 @@
                         <Grid TextElement.FontWeight="Bold"
                               TextElement.FontSize="14">
                             <Label Visibility="Collapsed"
-                                   Name="BrowsePathError">
-                                We could not identify a virtual environment in the selected path.
-                            </Label>
+                                   Name="BrowsePathError"
+                                   Content="{x:Static common:Strings.AddVirtualEnvironmentWindow_BrowsePathError}"/>
 
                             <Label Name="BrowsePathOrigPrefixError">
                                 <TextBlock TextWrapping="Wrap">
-                                    <Run>We could not find a base interpreter at </Run>
+                                    <!-- TODO: Localization make this a single string -->
+                                    <Run Text="{x:Static common:Strings.AddVirtualEnvironmentWindow_BrowsePathOrigPrefixError}"/>
                                     <Run Text="{Binding BrowseOrigPrefix}" />
```

Comment:
You should be able to specify the StringFormat parameter on the BrowseOrigPrefix binding to do string formatting. If it is harder then that, leave the TODO here.